src/components/dialogs/create-study-dialog.js

```diff
@@ -393,23 +442,24 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
             uploading: true,
         };
         createStudy(
-            caseExist,
             studyName,
             studyDescription,
-            caseName,
-            selectedFile,
+            caseName ?? tempCaseUuid,
             activeDirectory,
-            currentParameters && isParamsDisplayed
+            currentParameters &&
+                (isParamsDisplayed || isParamsCaseFileDisplayed)
```

Comment:
Same `isParamsDisplayed` and `isParamsCaseFileDisplayed` ?

Reply:
they not the same